Tests/MissingAttributeTests.swift

```diff
@@ -0,0 +1,249 @@
+//
+//  MissingAttributeTests.swift
+//  IBDecodableTests
+//
+//  Created by Yuta Saito on 2018/09/29.
+//
+
+import XCTest
+@testable import IBDecodable
+
+class MissingAttributeTests: XCTestCase {
+
+    lazy var cacheDirPath: URL = {
+        return self.fileManager
+            .urls(
+                for: .documentDirectory,
+                in: .userDomainMask
+            )
+            .first!
+            .appendingPathComponent("ibdecodable")
+    }()
+
+    let fileManager: FileManager = .default
+
+    func testRemoteResouces() {
+
+        func testFile(remoteURL: URL) {
+            do {
+                let content = try Data(contentsOf: remoteURL)
+                let parser = InterfaceBuilderParser()
+                let (_, indexer) = try parser.parseDocumentTraverse(data: content, type: XibDocument.self)
+                let result = indexer.dump()
+                print(result.description)
+                print("success: \(remoteURL)")
+            }
+            catch let error as InterfaceBuilderParser.Error {
+                switch error {
+                case .legacyFormat: return
+                case .macFormat: return
+                default:
+                    XCTFail("error: \(remoteURL): \(error)")
+                }
+            }
+            catch {
+                XCTFail("error: \(remoteURL): \(error)")
+            }
+        }
+
+        let expect = expectation(description: "Download")
+        fetchSample { (result) in
+            defer { expect.fulfill() }
+            switch result {
+            case .success(let response):
+                response.items.forEach { testFile(remoteURL: $0.rawURL) }
+            case .failed(let error):
+                XCTFail("\(error)")
+            }
+        }
+        waitForExpectations(timeout: 300) { (error) in
+            _ = error.map { XCTFail("\($0)") }
+        }
+    }
+
+    func fetchSample(handler: @escaping (Result<Github.Response>) -> Void) {
+        guard let token = ProcessInfo.processInfo.environment["GITHUB_ACCESS_TOKEN"] else {
+            XCTFail("You must define env var GITHUB_ACCESS_TOKEN")
+            return
```

Comment:
If no there is token, instead of `XCTFail`, we must call something like that `handler(.failed(Github.Error.missingAccessToken))`
The failure will be logged by the caller and `expect.fulfill()` will be called (so no long timeout before failing)

Reply:
You're right!